src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -405,8 +405,8 @@
       <XUnitLogCheckerHelixPath>$(XUnitLogCheckerHelixPath)XUnitLogChecker/</XUnitLogCheckerHelixPath>
 
       <XUnitLogCheckerArgs>$(_MergedWrapperRunScriptDirectoryRelative) $(_MergedWrapperName)</XUnitLogCheckerArgs>
-      <XUnitLogCheckerArgs Condition="'$(TestWrapperTargetsWindows)' != 'true'">$(XUnitLogCheckerArgs) %24HELIX_DUMP_FOLDER</XUnitLogCheckerArgs>
-      <XUnitLogCheckerArgs Condition="'$(TestWrapperTargetsWindows)' == 'true'">$(XUnitLogCheckerArgs) %25HELIX_DUMP_FOLDER%25</XUnitLogCheckerArgs>
+      <XUnitLogCheckerArgs Condition="'$(TestWrapperTargetsWindows)' != 'true'">$(XUnitLogCheckerArgs) %24%3F %24HELIX_DUMP_FOLDER</XUnitLogCheckerArgs>
```

Comment:
Would this better be in the helix command? As in save the result, not pass it in, and then exit with that if non-zero or with the fixer's exit otherwise?

Reply:
I agree with this. Something like:
```
// run test
set _saved_errorlevel=%errorlevel%
// run XunitLogChecker
if %errorlevel% NEQ 0 exit 1
if %_saved_errorlevel% NEQ 0 exit 1
exit 0
```
?

It's unfortunate that building a script in this helixpublishwitharcade.proj is comically complex.

Reply:
Makes sense to me.

Another question is whether we should preferentially return the test failing code, which would require saving the XUnitLogChecker code:
```
# Run test
test_exit_code=$?
# Run XUnitLogChecker
xunitlogchecker_exit_code=$?
# If the test failed, return that exit code
if [ $test_exit_code -ne 0 ]; then exit $test_exit_code; fi
if [ $xunitlogchecker_exit_code -ne 0 ]; then exit $xunitlogchecker_exit_code; fi
exit 0
```

Reply:
I think this is the best way to proceed. Thanks for your observations Bruce.